Jenkinsfile

```diff
@@ -237,13 +237,28 @@ pipeline {
                         dir("${BASE_DIR}") {
                           catchError(message: 'Beta stage', buildResult: 'SUCCESS', stageResult: 'UNSTABLE') {
                             dotnetWindows(){
-                              bat 'msbuild'
+                              bat label: 'Build', script: '.ci\\windows\\dotnet.bat'
                             }
                           }
                         }
                       }
                     }
                   }
+                  stage('Test') {
+                    steps {
+                      withGithubNotify(context: 'Test Docker msbuild - Windows', tab: 'tests') {
+                        cleanDir("${WORKSPACE}/${BASE_DIR}")
+                        unstash 'source'
+                        dir("${BASE_DIR}"){
+                          retry(3) {
+                            bat label: 'Build', script: '.ci/windows/dotnet.bat'
```

Comment:
is the build required to run twice? Or the current build output from the previous stage is sufficient as everything is already cached in the workspace?

On the other hand, what's the reason for the `retry` step? Did it fail previously to add that particular retry?